src/main/java/org/opentripplanner/routing/api/request/preference/AccessibilityPreferences.java

```diff
@@ -38,7 +41,16 @@ private AccessibilityPreferences(
     this.inaccessibleCost = Units.cost(inaccessibleCost);
   }
 
-  private static AccessibilityPreferences DEFAULT_UNSET = ofCost(NOT_SET, NOT_SET);
+  private AccessibilityPreferences(Builder builder) {
+    this.onlyConsiderAccessible = builder.onlyConsiderAccessible;
+    if (builder.onlyConsiderAccessible) {
```

Comment:
I think maybe we should allow to set cost when `onlyConsiderAccessible==true`, in that case this do not apply:

move `if` check to `Builder#build()` and return ONLY_CONSIDER_ACCESSIBLE, this make sure no extra instances is created. Then here you should assert that `builder.onlyConsiderAccessible == false` and throw `IllegalArgumentException` if not. If you want you may add this method to ObjectUtils:

```
  public static boolean requireFalse(Boolean value) {
    if (value == null || value) {
      throw new IllegalStateException(
        "Boolean field is required and expected to be false"
      );
    }
    return false;
  }
```

Reply:
I've updated `build()` to reuse `ONLY_CONSIDER_ACCESSIBLE` using `ofOnlyAccessible()` and `ofCost()`.

This removes the need for the above constructor.